docs/resources/synthetics_test.md

```diff
@@ -239,6 +259,12 @@ The following arguments are supported:
   - `params`: (Required) Parameters for the step as JSON string.
   - `allow_failure`: (Optional) Determines if the step should be allowed to fail.
   - `timeout`: (Optional) Used to override the default timeout of a step.
+- `variables`: (Optional) Array of variables used for the test.
+  - `type`: (Required) Type of browser test variable. Allowed enum values: "element","email","global","text"
+  - `name`: (Required) Name of the variable.
+  - `example`: (Optional) Example for the variable.
+  - `id`: (Optional) ID for the variable.
```

Comment:
This is actually only used (and required) in the case of using a variable of type `global`. In this case it is the ID of the global variable to use.

Reply:
Thank you for your comment. I update description.